.github/workflows/security_npm_dependency.yml

```diff
@@ -0,0 +1,12 @@
+name: Security npm dependency check
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "19 6 * * MON-FRI" # Every weekday
+jobs:
+  security-npm-dependency-check:
+    name: Project security npm dependency check
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_npm_dependency.yml@v0.7.1 # WORKFLOW_VERSION
```

Comment:
don't you want to float on minor version - why pin to patch version?  Maybe better to release `v1` and then float on major version instead